extra-devel/musl/autobuild/build

```diff
@@ -0,0 +1,18 @@
+abinfo "Building musl... "
+
+./configure --prefix=/usr/lib/musl \
```

Comment:
Unless this is not a standard GNU Autotools source, please use `ABTYPE=autotools` and rename this file to `beyond`, cut everything before line 14.

Reply:
This is not a standard GNU Autotools source, so `ABTYPE=autotools` won't work.

Reply:
Done.